instrumentation/aws-lambda-1.0/library/src/main/java/io/opentelemetry/instrumentation/awslambda/v1_0/TracingRequestWrapperBase.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.awslambda.v1_0;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+
+/**
+ * Base abstract wrapper for {@link TracingRequestHandler}. Provides: - delegation to a lambda via
+ * env property OTEL_LAMBDA_HANDLER in package.ClassName::methodName format
+ */
+abstract class TracingRequestWrapperBase<I, O> extends TracingRequestHandler<I, O> {
```

Comment:
I'm definitely not a fan of using an `O` as a type parameter. It looks way too much like a `0`.

Reply:
It's a copy from the root handler I (input), O (ouput). Would leave that to maintain consistency :)